Source/VCGeneration/Split.cs

```diff
@@ -87,6 +87,8 @@ void ObjectInvariant()
       readonly Dictionary<Block /*!*/, BlockStats /*!*/> /*!*/
         stats = new Dictionary<Block /*!*/, BlockStats /*!*/>();
 
+      public IEnumerable<AssertCmd> AssertCmds => blocks.SelectMany(block => block.cmds.OfType<AssertCmd>());
```

Comment:
I think there's already an `Asserts` property containing exactly this.

Reply:
Oh I see, it's in the newly VCsResult.. Let me check how I can use it there.

Reply:
Removed in the last commit